scripts/o3de/o3de/download.py

```diff
@@ -230,6 +306,9 @@ def add_parser_args(parser):
     parser.add_argument('-sar', '--skip-auto-register', action='store_true', required=False,
                             default=False,
                             help = 'Skip the automatic registration of new object download')
+    parser.add_argument('-fce', '--force-overwrite', action='store_true', required=False,
+                            default=False,
+                            help = 'Force overwrite the current object')
```

Comment:
I think we should go with the traditional `-f` / `--force` option the force argument.